### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by Skylan0916**
Fixed multiple missing determiner errors

### CODE REVIEW

**Comment by jklymak** [comment]
I don't think any of these are a particular improvement.  Suggest we close this.

File: README.rst [hunk 1]
```diff
@@ -106,10 +106,10 @@ Research notice
 ~~~~~~~~~~~~~~~
 
 Please note that this repository is participating in a study into
-sustainability of open source projects. Data will be gathered about this
+the sustainability of open source projects. Data will be gathered about this
 repository for approximately the next 12 months, starting from June 2021.
 
-Data collected will include number of contributors, number of PRs, time taken
+Data collected will include the number of contributors, number of PRs, time taken
```
Comment (QuLogic) [inline README.rst:112]: Should have `the` on `number of PRs` as well if you're going to add it.

**Comment by story645** [comment]
I agree w/ @QuLogic that  the articles are missing from  the last two examples and that there's no article for the first.

File: README.rst (hunk 1 above)
Comment (tacaswell) [reply README.rst:112]: We should not edit the text in this section as it is the verbatim wording from @yochannah for the research notice via f0dcb6e7b94e62a0983e804d4c84df83f87185a9 / https://github.com/matplotlib/matplotlib/pull/20362 .  This text came in as part of the ethics requirements for a research study and we should not risk invalidating that on minor grammar changes.

**Comment by tacaswell** [comment]
Thank you for your work on this @Skylan0916 , however I am going to close this without merging.  There is consensus one part of the change is not an improvement and the other part is text that should stay verbatim as it is.

I hope we hear from you again!

**Pull request closed without merging**